packages/docs/developer-resources/start.md

```diff
@@ -1,11 +1,30 @@
 # Start
 
-The following examples will serve as a guide to understand the Celo SDK:
+The following examples and guides will help you start building on Celo.
 
-* The [Sending CELO](walkthroughs/hellocelo.md) walkthrough will guide you through the basics of connecting to Celo networks using ContractKit.
+## Backend / command line interface
 
-* The [Deploy a Contract (local node)](walkthroughs/hellocontracts.md) walkthrough will guide you through the basics of how to deploy your own smart contracts from a local Celo node.
+- [Sending CELO](walkthroughs/hellocelo.md)
 
-* The [Deploy a Contract (Forno)](walkthroughs/hello-contract-remote-node.md) demonstrates how to deploy smart contracts with [Truffle](https://www.trufflesuite.com/truffle) and [Forno](forno/), a remote node service provider.
+  - The sending CELO walkthrough will guide you through the basics of connecting to Celo networks using ContractKit and sending a test transaction.
 
-* The [DappKit Truffle Box](walkthroughs/hello-mobile-dapp.md) walkthrough will help you create a simple mobile dApp using dAppKit and the React Native Expo framework.
+- [Deploy a Contract (local node)](walkthroughs/hellocontracts.md)
+
+  - Our Deploy a Contract walkthrough will guide you through the basics of how to deploy your own smart contracts from a local Celo node. In this tutorial you'll deploy a contract with a simple API for writing and reading names.
+
+- [Deploy a Contract (Forno)](walkthroughs/hello-contract-remote-node.md)
+
+  - Our second Deploy a Contract tutorial demonstrates how to deploy smart contracts with [Truffle](https://www.trufflesuite.com/truffle) and [Forno](forno/), a remote node service provider.
+
+## React Native
```

Comment:
Suggest updating this to `Mobile DApps` rather than `React Native`. We can add mobile tutorials here that aren't limited to React Native.